setup.sh

```diff
@@ -0,0 +1,7 @@
+fauna query --input seed/categories.fql --secret "$FAUNA_SECRET"
+echo -e "\n\n"
+fauna query --input seed/customers.fql --secret "$FAUNA_SECRET"
+echo -e "\n\n"
+fauna query --input seed/products.fql --secret "$FAUNA_SECRET"
+echo -e "\n\n"
+fauna query --input seed/orders.fql --secret "$FAUNA_SECRET"
```

Comment:
You probably didn't see it, but I put a new file `test/setup.sh` that somewhat duplicates this file, I think we need to rename that file, or this file. Maybe we could rename this one `seed.sh` and put this script along with `setup.sh` and `validate.sh` in a directory called `scripts`?

Reply:
Thanks! I did the following:

* Renamed this file to `seed.sh` and placed it in `/scripts`.
* Moved `/test*` files over to `/scripts`.
* Updated references in `test.yml`

There some additional work needed to update `test.yml` to the CLI v4 beta. Once that's done, we can just call `seed.sh` from `setup.sh`. However, that's a bit beyond the scope of this PR.

Feel free to push changes to this PR/branch. Appreciate you taking a look!